### PR TITLE
Build busybox for amd64/arm64/armhf

### LINKED ISSUE #1: Include busybox for amd64 arch
For using with qemu etc.

### PR DESCRIPTION
**Pull request by anupcshan**
Fixes #1 

- Upgrade to busybox 1.34.1
- Build tooling to make it easy to reproduce the build or upgrade/modify busybox in the future
- Switch to go:embed for storing files in binaries

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+RUN mkdir /src /bins && curl -s https://busybox.net/downloads/busybox-1.34.1.tar.bz2 | tar --strip-components=1 -C /src -xjf -
+
+COPY busybox.config /src/.config
+
+ENV SOURCE_DATE_EPOCH 1600000000
+
+RUN cd /src && make CROSS_COMPILE=aarch64-linux-gnu- ARCH=arm64 -j$(nproc) busybox && cp /src/busybox /bins/busybox.arm64
+RUN cd /src && make CROSS_COMPILE=arm-linux-gnueabihf- ARCH=arm -j$(nproc) busybox && cp /src/busybox /bins/busybox.arm
+RUN cd /src && make clean && make -j$(nproc) busybox && cp /src/busybox /bins/busybox.x64
```
Comment (stapelberg) [inline Dockerfile]: Can we name this one amd64 for consistency with Go build tags?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+third_party/busybox/busybox.x64: Dockerfile busybox.config
+	docker build --rm -t serial .
+	docker run --rm -v $$(pwd)/third_party/busybox:/tmp/bins serial cp -r /bins/ /tmp/
```
Comment (stapelberg) [inline Makefile]: Any reason not to keep the version number in the directory name here?

**Review by stapelberg — COMMENTED** [review]
Thanks for your PR!

File: Dockerfile (hunk 1 above)
Comment (anupcshan) [reply Dockerfile]: Will do

File: Makefile (hunk 2 above)
Comment (anupcshan) [reply Makefile]: I used a non-versioned location mostly to reduce the number of places to update when a version upgrade happens. Happy to do it either way.

**Review by stapelberg — APPROVED** [review]

**Comment by stapelberg** [comment]
Thanks very much for your PR!

**Pull request merged**